apps/web/src/pages/Users/UserInformationPage/components/CandidateStatusSection.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { useIntl } from "react-intl";
+
+import { Heading, Well } from "@gc-digital-talent/ui";
+
+import { JobLookingStatus } from "~/api/generated";
+
+import { UserInformationProps } from "../types";
+import AddToPoolDialog from "./AddToPoolDialog";
+import PoolStatusTable from "./PoolStatusSection";
+
+const CandidateStatusSection = ({ user, pools }: UserInformationProps) => {
+  const intl = useIntl();
+
+  return (
+    <>
+      <Heading level="h4" data-h2-margin="base(x2, 0, x1, 0)">
+        {intl.formatMessage({
+          defaultMessage: "Personal status",
+          id: "4N6O+3",
+          description:
+            "Title of the 'Personal status' section of the view-user page",
+        })}
+      </Heading>
+      <Well>
+        {user.jobLookingStatus === JobLookingStatus.ActivelyLooking &&
+          intl.formatMessage({
+            defaultMessage:
+              "<heavyPrimary>Active</heavyPrimary> - Wants to be contacted for job opportunities",
+            id: "SOZVtc",
+            description:
+              "Text in view user page saying they currently have the 'Active' status, ignore things in <> tags please",
```

Comment:
Removed in [remove instructions from message description](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5707/commits/1e5709c633a7ec418e2e90842ac87df02c23cf61)

I found a few more references but not sure if I should remove them from files unrelated to this PR 🤔

Reply:
When any part of `defaultMessage` or `description` is changed, so does the `id`. You'll need to save the original English strings to generate the new IDs where they appear and extract and merge again.

Reply:
**_Very_** confused how it let me save and commit those files with old IDs 😖 Fixed in [update message ids](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5707/commits/8e6f831734bba57679829a438b220c2bc943c61f)

Reply:
I am guessing because the IDs remained the same and the check is an eslint rule that would just highlight the problem and then there were no GitHub checks run (importantly, the lint workflow) because the branch has unresolved conflicts?